Intersect.Server/Database/Item.cs

```diff
@@ -122,6 +122,18 @@ public bool TryGetBag(out Bag bag)
         {
             bag = Bag;
 
+            if (bag != null)
+            {
+                //Remove any items from this bag that have been removed from the game
+                foreach (var itm in bag.Slots)
+                {
+                    if (itm.ItemId != Guid.Empty && ItemBase.Get(itm.ItemId) == null)
+                    {
+                        itm.Set(new Item());
```

Comment:
Just wondering - in other places I've seen `Item.Set(Item.Empty)` instead - which do we prefer?